docs/user/reporting/index.asciidoc

```diff
@@ -67,6 +67,8 @@ NOTE: When you create a dashboard report that includes a data table or saved sea
 
 . To view and manage reports, open the main menu, then click *Stack Management > Reporting*.
 
+NOTE: Reports stored in Elasticsearch are managed by the `kibana-reporting` index lifecycle management (ILM) policy. By default reports are stored forever. {ref}/index-lifecycle-management.html[Learn more] about ILM policies.
```

Comment:
It's not clear from the rest of the page that reports are stored in Elasticsearch. The current copy implies you can store reports outside of Elasticsearch. I tried to clarify that here.

The first sentence is passive voice, but I think that's the best way to state this.
```suggestion
NOTE: Reports are stored in {es} and managed by the `kibana-reporting` {ilm}
({ilm-init}) policy. By default, the policy stores reports forever. To learn
more about {ilm-init} policies, refer to the {es}
{ref}/index-lifecycle-management.html[{ilm-init} documentation].
```

Reply:
Got it, yeah in future we will enable you to store reports outside of Elasticsearch, so that was intended, but for not it is not possible. We'll need to update this copy in future! 👍🏻